frontend/lib/util/env.ts

```diff
@@ -0,0 +1 @@
+export const isDevelopment = process.env.NODE_ENV === "development";
```

Comment:
there are more process.env vars used throughout the app I think, could you add those as well? (i like this)

Reply:
I have only found them in configuration files - moving those to the codebase might be problematic. So for now this is the only one used at compile/runtime.